### ISSUE TITLE
Liquid suffixes on style assets

### ISSUE THREAD

**Issue by t-kelly** [Shopify/slate#230]
<a href="https://github.com/NathanPJF"><img src="https://avatars2.githubusercontent.com/u/6284114?v=4" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [NathanPJF](https://github.com/NathanPJF)**
_Thursday May 04, 2017 at 22:42 GMT_
_Originally opened as https://github.com/Shopify/slate-tools/issues/57_

----

### Problem

Any `*.css` or `*.scss` file in the `src/` directory is always converted to a `.liquid` file - regardless whether the files actually contains any Liquid.   There's no way to opt-out of a `*.liquid` file in the build process.  

Problem with the Liquid file is that it takes longer to compile in the online editor vs. a "regular" SCSS or CSS file.  Larger files will fail to compile in the Editor.

### Scope

This came to light in this issue: https://github.com/Shopify/slate/issues/133

Importing vendor frameworks into `src/styles/theme.scss` increases the file size of the `theme.scss.liquid` file sent to `dist/` and uploaded to Shopify.

Shopify has a 15 second timeout on Imagery currently.  So larger `*.scss.liquid` files risk never being compiled.  We can speed up the file compilation by not appending `*.liquid` if it's not needed.

### Replication steps

Shopifolk can find reproduction steps in this private repo: https://github.com/Shopify/shopify/issues/112863

TL;DR: A Slate theme using [Semantic UI](https://github.com/Semantic-Org/Semantic-UI-CSS) CSS as a vendor import in `src/styles/theme.scss`.  

```
/*================ VENDORS ================*/
@import url('vendor/semantic.css');
```

Times out in compilation:

![](http://take.ms/utWob)

### More Information

In the gulp tasks, `processCss()` is replacing all `*.scss` with `*.scss.liquid` : https://github.com/Shopify/slate-tools/blob/e7bdfe3f77a5610d29590c5d26c82bb76c8a10ca/src/tasks/build-css.js#L21

We _don't_ do this liquid extension for JS files - as seen in `[processThemeJS()](https://github.com/Shopify/slate-tools/blob/e7bdfe3f77a5610d29590c5d26c82bb76c8a10ca/src/tasks/build-js.js#L11)`.  Likely reason: we build for the first-party themes for the editor and strive to keep Liquid out of our JS files. 

Potential idea: Remove `.pipe(extReplace('.scss.liquid', '.scss'))` from the function.  If developers need Shopify to compile SCSS with Liquid, then they will have to write `*.liquid` on the end of the file name.

**Comment by t-kelly** [comment]
<a href="https://github.com/lixonic"><img src="https://avatars0.githubusercontent.com/u/934220?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [lixonic](https://github.com/lixonic)**
_Tuesday Jul 18, 2017 at 17:01 GMT_

----

any update ?

**Comment by t-kelly** [comment]
Closing in preference to #258
